source/gems/dry-container/registry-and-resolver.html.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Registry &amp; Resolver
+layout: gem-single
+---
+
+### Register options
+
+#### `call`
+
+This option register container wrapped (or not) to proc:
+
+```ruby
+container = Dry::Container.new
+container.register(:key_1, call: false) { "Integer: #{rand(1000)}" }
+container.register(:key_2, call: true)  { "Integer: #{rand(1000)}" }
+
+container.resolve(:key_1) # => <Proc:0x007f98c90454c0@dry_c.rb:23>
+container.resolve(:key_1) # => <Proc:0x007f98c90454c0@dry_c.rb:23>
+
+container.resolve(:key_2) # => "Integer: 157"
+container.resolve(:key_2) # => "Integer: 713"
+```
+
+#### `memoize`
+
+This option register memoized (or not) container:
```

Comment:
Think this would be clearer as `This boolean option determines whether or not the registered item should be memoized on the first invocation, i.e.` or similar